### PR TITLE
Allow to specify a specific version of Varnish

### PR DESCRIPTION
**Pull request by d3clan**
I have only verified that this works on CentOS release 6.5.

### CODE REVIEW

**Comment by thanandorn** [comment]
@d3clan 

IMO, variable name should be $package_ensure because it'll be used for ensure directive of the package resource.

We should not initial a specific version by default. The value should just be 'present' in params.pp.

**Comment by d3clan** [comment]
Updated the variable name and defaulted to 'present', as per comments.

File: manifests/init.pp [hunk 1]
```diff
@@ -52,6 +52,7 @@
   $storage_size = $varnish::params::storage_size,
   $varnish_version = $varnish::params::varnish_version,
   $vcl_reload = $varnish::params::vcl_reload,
+  $specific_version = $varnish::params::package_ensure,
```
Comment (thanandorn) [inline manifests/init.pp]: Please correct variable name so it can be referred in `class { 'varnish::install'`

:)

File: manifests/install.pp [hunk 2]
```diff
@@ -4,6 +4,6 @@
   include varnish::params
 
   package { $varnish::params::package_name:
-    ensure => present,
+    ensure => $varnish::params::package_ensure,
```
Comment (thanandorn) [inline manifests/install.pp]: This should just be `ensure => $varnish::package_ensure` so we can specify version from `class { 'varnish': `.

:)

**Pull request merged**